src/main/java/org/jqassistant/contrib/plugin/asciidocreport/plantuml/PlantUMLRenderer.java

```diff
@@ -222,20 +224,38 @@ private String getNodeId(Node node) {
      *
      * @param plantUML
      *            The diagram.
+     * @param format
+     *            The target format.
      * @param file
      *            The {@link File}.
      */
-    private void renderDiagram(String plantUML, File file) {
+    private void renderDiagram(String plantUML, File file, FileFormat format) {
         SourceStringReader reader = new SourceStringReader(plantUML);
         try {
             LOGGER.info("Rendering diagram '{}' ", file.getPath());
             try (FileOutputStream os = new FileOutputStream(file)) {
-                reader.outputImage(os, new FileFormatOption(PLANTUML_FILE_FORMAT));
+                reader.outputImage(os, new FileFormatOption(format));
             }
 
         } catch (IOException e) {
             throw new IllegalStateException("Cannot create component diagram for file " + file.getPath());
         }
     }
 
+    /**
+     * Trys to parse a given String to a PlantUML-FileFormat
+     *
+     * @param format
+     *            The {@link FileFormat} as string.
+     * @return The matching {@link FileFormat} or DEFAULT_PLANTUML_FILE_FORMAT if none matches.
+     */
+    private FileFormat toFileFormat(String format) {
```

Comment:
I would prefer if an error/exception would be reported if a non-supported file format is specified